AIO-Tools/MainMenu.cs

```diff
@@ -858,12 +858,14 @@ private void SubPathsTrue_SaveFalse()
         {
             Logging.SpecificLog("SubPathsTrue_SaveFalse", "DownloadStates");
             // IF no Select download Patch
-            FolderBrowserDialog folderDlg = new FolderBrowserDialog();
-            DialogResult result = folderDlg.ShowDialog();
-            if (result == DialogResult.OK & result != DialogResult.Cancel)
+            CommonOpenFileDialog folderDlg = new CommonOpenFileDialog
```

Comment:
If you do this to mainmenu, why didnt you do that to other?

Reply:
I made this same change to all folder browsers, this version is just mush more usable.